Automattic/jetpack
# E2E Tests: Add Scheduled job for a Pre-Release Gutenberg block tests

#### Changes proposed in this Pull Request:

* Adds a scheduled E2E job to run block tests against pre-release Gutenberg
* Re-organize scripts a bit 

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






There is no easy way to test it, unfortunately. To properly test it:

- Checkout this branch and run `GUTENBERG=latest ./bin/env.sh start` to make sure the plugin setup works as expected
- Create an action in your personal test repo. Something like https://github.com/brbrr/action-test/blob/master/.github/workflows/test-action.yml
- Commit and wait to see some scheduled actions to be triggered
- OR check my test repo, where I did the tests: https://github.com/brbrr/action-test
